docs/MO_DG/prepare_model/FP16_Compression.md

```diff
@@ -0,0 +1,21 @@
+# Compression of a Model to FP16 {#openvino_docs_MO_DG_FP16_Compression}
+
+Model Optimizer can convert all floating point weights to `FP16` data type. The resulting IR is called
+compressed `FP16` model.
+
+To compress the model use the `--data_type` option like this:
+
+```
+mo --input_model /path/to/model --data_type FP16
+```
+
+> **NOTE**: Using `--data_type FP32` will give no result and will not force `FP32` 
+> precision in the model. If the model was `FP16` it will have `FP16` precision in IR as well.
+
+The resulting model will occupy about twice as less space in the file system, but it will have slightly worse accuracy
+on hardware that support `FP16` inference, although for the majority of models accuracy degradation is negligible. 
```

Comment:
1. I'd say that compressed IR **may** have some accuracy drop (we can't be sure that accuracy is slightly worse for all models)
2. accuracy drop may occur for ANY hw due to weights compression itself, so I think we shouldn't mention HW capabilities here